flickr.py

```diff
@@ -25,19 +25,22 @@ def params_unique_combination(baseurl, params_d, private_keys=["api_key"]):
             res.append("{}-{}".format(k, params_d[k]))
     return baseurl + "_".join(res)
 
-def search_flickr_by_tags(tags):
+def search_flickr(tag, key, value):
```

Comment:
I changed the argument to take one argument, and I was hoping to add a function in between to categorize what type of argument one is passing